### PR TITLE
Load kubernetes configuration

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: deployment/config.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+// KubernetesConfig returns Kubernetes configuration for native Kubernetes
+// environment
+func KubernetesConfig() (*restclient.Config, error) {
+	token, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	if err != nil {
+		return nil, err
+	}
+	return &restclient.Config{
+		Host:        "http://" + os.Getenv("KUBERNETES_SERVICE_HOST") + ":" + os.Getenv("KUBERNETES_PORT_443_TCP_PORT"),
```
Comment (bobbytables) [inline deployment/config.go]: We _really_ should not be using environment variables from inside packages, we should pass in the values to this function. All values should be passed from environment variables from the CLI `main` package, not in the package itself.

Comment (macat) [inline deployment/config.go]: Thanks, we are going to do this.

**Comment by poopoothegorilla** [comment]
looks good so far :+1:

**Comment by bobbytables** [comment]
:+1:

**Pull request merged**